### DISCUSSION TITLE
Multiple WebView2 instances same parent window

### DISCUSSION THREAD

**Discussion by ealva** [MicrosoftEdge/WebView2Feedback#2135] [Q&A] [answered]
We already have an implementation that's similar to this:

```
ParentWnd (hosts WebView2 #1)
  |
  +-ChildWnd (hosts WebView2 #2)
```

`ChildWnd` is used to render offline pages and is shown / hidden in certain scenarios (e.g., splash screen).

We're wondering if it's OK / safe / advisable to have `WebView2 #2` as a child of `ParentWnd` instead and if this saves us anything significant (or have potential gotchas).  (This in Win32, btw).

**Comment by victorthoang** [comment, marked as answer]
hello @ealva 

Are you still looking for an answer to your question? If you still need a response, we can turn this into an Issue with a Question tag (which is typically how we address questions and bugs.

The discussion thread might be more useful for off topic questions or non-issue related inquiries!

Let me know if you'd like me to create an issue from this discussion, or if you'd prefer to add it. Thanks!

**Comment by ealva** [comment]
Thanks @victorthoang. I forgot about this already. This is resolved.

**Comment by ajtruckle** [comment]
Then why not put your solution for others benefit?
________________________________

**Comment by ealva** [comment]
Thank you @ajtruckle. Trying to remember the details, and this is more of a question for us early on.

We settled on a single window solution and toggling WebView2's `IsVisible` property to show / hide instances. This worked pretty well (for us, at least) and with less complexity than having to manage multiple window hosts.

Hope this helps others facing the same questions.
